documentation/book/assembly-oauth-config.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-oauth-strimzi-config-{context}']
+= Configuring {oauth} authentication
+
+{oauth} is used for interaction between Kafka clients and {ProductName} components.
+
+In order to use {oauth} for {ProductName}, you must:
+
+. xref:proc-oauth-server-config-{context}[Deploy an authorization server and configure the deployment to integrate with Strimzi]
+. xref:proc-oauth-broker-config-{context}[Deploy or redeploy the Kafka cluster with Kafka broker listeners configured to use {oauth}]
```

Comment:
"redeploy" ... it's not really about re-deploy which sounds to me like deleting and deploying the cluster again. Actually, it's about updating the Kafka configuration with updated listeners in order to have the CO doing the rolling update.

Reply:
Changed to 'update'